packages/core/client/src/schema-component/antd/icon-picker/__tests__/icon-picker.test.tsx

```diff
@@ -47,4 +47,15 @@ describe('IconPicker', () => {
     await userEvent.click(icons[1]);
     expect(screen.queryAllByRole('img').length).toBe(0);
   }, 300000);
+
+  it('should filter the displayed icons when changing the value of search input', async () => {
+    const { container } = render(<App />);
+
+    const button = container.querySelector('button') as HTMLButtonElement;
+    await userEvent.click(button);
+
```

Comment:
<img width="1179" alt="image" src="https://github.com/nocobase/nocobase/assets/3250534/b47fe57a-56e8-45d4-b2d9-2c5c5ea389ff">

Try to add `await waitFor()` here, waiting util the IconPicker component rendered.

Reply:
I just tried to adapt the steps from existing code, and not see the actual result, because I've trouble running vitest atm on this windows machine.

Reply:
How did you run vitest and do you have the error messages or screenshots? I may help to fix it.

Reply:
@2013xile I've got them running by stoping serving with `yarn dev` first, thanks.

However, I'm having problems fixing the tests because I can't seem to get the searchbar in the popover selected. Hope you can help me here